examples/visualization/plot_report_slider.py

```diff
@@ -0,0 +1,25 @@
+from mne.report import Report
+from mne.datasets import sample
+from mne import read_evokeds
+from mne.utils import _TempDir
+
+
+report = Report()
+path = sample.data_path()
+fname = path + '/MEG/sample/sample_audvis-ave.fif'
+tempdir = _TempDir()
+report_path = tempdir + '/report.html'
+
+# Load the evoked data
+evoked = read_evokeds(fname, condition='Left Auditory',
+                      baseline=(None, 0), verbose=False)
+evoked.crop(0, .2)
+times = evoked.times[::4]
+# Create a list of figs for the slider
+figs = list()
+for time in times:
+    figs.append(evoked.plot_topomap(time, vmin=-300, vmax=300,
+                                    res=100, show=False))
+    plt.close()
+report.add_slider_to_section(figs, times, 'Evoked Response')
+report.save(report_path, open_browser=True)
```

Comment:
do we have a `report.view()`? We should add one. It can dump the report to a temporary file, then call something like `import webbrowser; webbrowser.open_new_tab(filename)`